include/simple2d.h

```diff
@@ -130,13 +135,16 @@ typedef struct {
   int direction;
   int axis;
   int value;
+  int64_t finger_id;
+  int64_t touch_id;
```

Comment:
Instead of `int64_t`, do you think these should be of type `SDL_TouchID` and `SDL_FingerID`, respectively?